docs/Contributors_Guide/update_database_schema.rst

```diff
@@ -0,0 +1,88 @@
+
+
+*************************************
+Update the Database Schema
+*************************************
+
+Background
+===========
+
+The METdataio METdbLoad module is used to load input data into the METviewer database. It is important that
+the database schema is updated when the following conditions apply:
+
+* The MET tool has introduced a *new linetype*
+* The MET tool has *modified one or more existing linetype(s)*
+* A new METplus *coordinated release*
+  (i.e. x.y METplus release with all components: METplus, MET, METviewer,
+  METdataio, METcalcpy, METplotpy)
+* A new *bugfix* or *beta release* of the METplus components: METdataio and MET
+
+Steps
+=====
+
+1. Create a Github issue in the METdataio repository.
+
+*  Click on the **Issues** tab at the top of the `METdataio Repository page <https://github.com/dtcenter/METdataio>`_
+*  Click on the green **New issue** button on the top right of the issues page.
+*  Click on the green **Get started** button corresponding to the **Task** template
+*  Assign the appropriate settings under
+   the *Assignees*, *Labels*, *Projects* and *Milestones* links to the right side of the Github issues
+   page.
+*  Fill in relevant information under the **Add a description** window
+
+
+2. Create a feature branch corresponding to this Github issue in the METdataio repository:
+
+.. code-block:: ini
+
+    feature_xyz_update_db_schema
+
+where **xyz** corresponds to the Github issue number and is branched from the *develop* branch
+
+.. code-block:: ini
+
+   git checkout develop
+   git checkout -b feature_xyz_update_db_schema
+
+3. Create an **update_for_x_y.sql** file in the $BASE_DIR/METdbLoad/sql/updates directory
+
+   * $BASE_DIR corresponds to the directory where the METdataio source code resides
+   * **x_y** corresponds to the METviewer version (e.g 5_1 for 5.1)
```

Comment:
I would use METplus Analyses tools version instead of METviewer to be more generic

Reply:
OK, I changed it to:
3. Create an **update_for_x_y.sql** file in the $BASE_DIR/METdbLoad/sql/updates directory

   * $BASE_DIR corresponds to the directory where the METdataio source code resides
   * **x_y** corresponds to the METplus Analysis version (e.g 5_1 for 5.1)